docs/api/endpoints.md

```diff
@@ -270,3 +270,173 @@ Status: 200 OK
     "is_superuser": false
 }
 ```
+
+# Proposals
+
+## Create Proposal
+
+```
+POST /api/proposals
+```
+
+**Parameters**
+
+Name               | Data type     | Required | Description
+-------------------|---------------|----------|---------------------
+id                 | UUID          | false    | Unique ID for the proposal
+title              | text          | true     | Title of proposal
+speaker            | text          | false    | Speaker for the talk
+kind               | text          | true     | Type of proposal like talk, dev sprint, workshop
```

Comment:
like means nothing. Be specific. Say these are the options and please put options in backquotes for markdown format to recognize them and display them accordingly.

This change should be done elsewhere too, where docs are done.